sysutils/opentofu/Portfile

```
@@ -0,0 +1,144 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           select 1.0
+
+name                opentofu
+categories          sysutils
+maintainers         {@graywolf wolfsden.cz:~+macports} openmaintainer
+license             MPL-2
+
+homepage            https://opentofu.org/
+
```

```suggestion

conflicts           tenv

```

@graywolf A new port [`tenv`](https://ports.macports.org/port/tenv) was recently added. Since it installs a `tofu` binary in `${prefix}/bin`, it will conflict with this OpenTofu port. It currently already [conflicts with the Terraform port](https://github.com/macports/macports-ports/blob/67821ec5d8849f94e3562c8fd681ec6e58afaa2b/sysutils/tenv/Portfile#L17) since it installs a `terraform` binary as well.